### ISSUE TITLE
How to activate intellisense in Console Application?

### ISSUE THREAD

**Issue by shyamal890** [Antaris/RazorEngine#398]
I tried following instructions [here](https://antaris.github.io/RazorEngine/IntellisenseAndResharper.html) but as soon as I type `@using RazorEngine` it gives me an error namespace not found. How can I get intellisense support with RazorEngine nuget package installed?

EDIT: Following this [RazorEngine configuration](https://github.com/NancyFx/Nancy/wiki/Razor-View-Engine#configuring-razor-using-appconfig-or-webconfig) link I created a Views Folder in root of console application. Created a new App.Config under Views folder with the following code in it:

```
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
  <configSections>
    <section name="razor" type="Nancy.ViewEngines.Razor.RazorConfigurationSection, Nancy.ViewEngines.Razor" />
  </configSections>
  <razor disableAutoIncludeModelNamespace="false">
    <namespaces>
      <add namespace="my.namespace" />
    </namespaces>
  </razor>
</configuration>
```
Still intellisense doesn't work.

**Comment by AllanXu49** [comment]
Your EXE and DLLs are in the bin\Debug\. You need to manually copy them to BIN folder where Razor IntelliSense is looking at. 

I had the very same issue, posted on stack overflow and answered myself.

See here:

http://stackoverflow.com/questions/38773186/razorengine-and-visual-studio-2015-inellisense

"Your project output path is set to bin\ instead of bin\Debug\ and bin\Release. another possible solution is to copy RazorEngine.dll and System.Web.Razor.dll to bin."

**Comment by matthid** [comment]
Related https://github.com/Antaris/RazorEngine/issues/368
Thanks @AllanXu49 for answering this. I think this has been solved (or is tracked in #368).

**Comment by HumanAftrAll** [comment]
I've gone through the same problem and adding a `\Views\Web.config` file to the console application root folder did the trick. Now I can reference `RazorEngine` and my own `Namespace`.

Although this problem has been solved, IDK why I can't use the `inherits` nor the `model` keywords on my template.
I've changed the compilation folder from `\bin\Debug\` and `\bin\Release\ `to just `\bin\`, but it did not work.

I'm using: 
- .NET Framework v4.7.2
- RazorEngine v3.10.0
- Microsoft.AspNet.Razor v3.2.7

@matthid what other info can I provide so you can, hopefully, help me?

Thanks!
